portfolio/src/main/java/com/google/sps/model/Comment.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.model;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+public class Comment {
+  private long id;
+  private String content;
+  private String userName;
+  private Date creationTime;
+  private long likeCount;
+
+  private Comment(long id, String userName, String content, Date creationTime, long likeCount) {
+    this.id = id;
+    this.content = content;
+    this.userName = userName;
+    this.creationTime = creationTime;
+    this.likeCount = likeCount;
+  }
+
+  public static Entity toEntity(String userName, String content) {
```

Comment:
just FYI, this seems questionable from the design perspective. I think a CommentHelper type of class would be more suitable for a method like this because you are not really using Comment object here. That said, I think it is OK to keep - the class and use case are so simple that it is fine as is.

Reply:
I think object makes Json conversion easier. But, helper class does fit better in here because Comment object is mostly used for static methods.